newrelic-agent/src/main/java/com/newrelic/agent/TimedTokenSet.java

```diff
@@ -37,14 +36,13 @@ public TimedTokenSet(int timeOut, TimeUnit unit, final ExpirationService expirat
         // onRemoval happens immediately after put, so we can hit onRemoval logic before getToken() even finishes
         long timeOutMilli = TimeConversion.convertToMilliWithLowerBound(timeOut, unit, 250L);
 
-        activeTokens = CacheBuilder.newBuilder()
-                .concurrencyLevel(8)
+        activeTokens = Caffeine.newBuilder()
+                .initialCapacity(8)
```

Comment:
For context see the discussion @ https://github.com/newrelic/newrelic-java-agent/pull/286#discussion_r621944669. We should consider dropping this line.

Reply:
We are going to keep this change as is with capacity set to 8.  I'm creating a research spike to unblock getting this merged and investigate whether resizing makes sense.